src/foundation.cs

```diff
@@ -8194,6 +8194,18 @@ interface NSInputStream {
 
 	}
 
+	delegate bool EnumerateLinguisticTagsEnumerator (NSString tag, NSRange tokenRange, NSRange sentenceRange, ref bool stop);
```

Comment:
`EnumerateLinguisticTagsEnumerator` I think we should prefix this with `NS` to match the other types

Reply:
Ok, I have no preference and will help users.